lib/class-wp-theme-json-gutenberg.php

```diff
@@ -333,6 +342,10 @@ class WP_Theme_JSON_Gutenberg {
 			'style'  => null,
 			'width'  => null,
 		),
+		'shadow' 						=> array(
+			'palette'          => null,
+			'defaultPalette'   => null,
```

Comment:
I have tested setting `shadow.defaultPalette` to `false` in the `theme.json`, but it does not have any effect in the UI. Example of how gradients use it at https://github.com/WordPress/gutenberg/blob/61adfcc5b9a57febdce9ba23743b183b10a4b3f8/packages/edit-site/src/components/global-styles/gradients-palette-panel.js#L44

Reply:
Fixed here [e25741c](https://github.com/WordPress/gutenberg/pull/46502/commits/e25741ca3a7b3e0361e608195d3ec554b4c2325c)